TicketMS/Controllers/OrderController.cs

```diff
@@ -11,49 +12,60 @@ namespace TicketMS.Controllers
     public class OrderController : ControllerBase
     {
         private readonly IOrderRepository _orderRepository;
+        private readonly IMapper _mapper;
 
-        public OrderController(IOrderRepository orderRepository)
+        public OrderController(IOrderRepository orderRepository, IMapper mapper)
         {
             _orderRepository = orderRepository;
+            _mapper = mapper;
         }
 
         [HttpGet]
-        public ActionResult<List<Order>> getAllOrders()
+        public async Task<ActionResult<List<Order>>> getAllOrders()
```

Comment:
All methods name in C# must start with uppercase letter

Reply:
Done